code/game/mecha/micro/micro.dm

```diff
@@ -0,0 +1,297 @@
+					melee_can_hit = 0
+					if(do_after(melee_cooldown))
+						melee_can_hit = 1
+					break
+	return
+
+/*
+/obj/mecha/micro/proc/mega_shake(target)
+	if(!istype(target, /obj) && !istype(target, /mob)) return
+	if(istype(target, /mob))
+		var/mob/M = target
+		M.make_dizzy(3)
+		M.adjustBruteLoss(1)
+		M.updatehealth()
+		for (var/mob/V in viewers(src))
+			V.show_message("[src.name] shakes [M] like a rag doll.")
+	return
+*/
+
+/*
+	if(energy>0 && can_move)
+		if(step(src,direction))
+			can_move = 0
+			spawn(step_in) can_move = 1
+			if(overload)
+				energy = energy-2
+				health--
+			else
+				energy--
+			return 1
+
+	return 0
+*/
+/*
+/obj/mecha/micro/hear_talk(mob/M as mob, text)
+	..()
+	if(am && M==occupant)
+		if(findtext(text,""))
+			sam()
+	return
+
+/obj/mecha/micro/proc/sam()
+	if(am)
+		var/window = {"<html>
+							<head>
+							<style>
+							body {background:#000;color: #00ff00;font-family:"Courier",monospace;font-size:12px;}
+							#target {word-wrap: break-word;width:100%;padding-right:2px;}
+							#form {display:none;padding:0;margin:0;}
+							#input {background:#000;color: #00ff00;font-family:"Courier",monospace;border:none;padding:0;margin:0;width:90%;font-size:12px;}
+							</style>
+							<script type="text/javascript">
+							var text = "SGNL RCVD\\nTAG ANL :: STTS ACCPTD \\nINITSOC{buff:{128,0,NIL};p:'-zxf';stddev;inenc:'bin';outenc:'plain'}\\nSOD ->\\n0010101100101011001000000101010001101000011010010111001100100000011011010110000101100011011010000110100101101110011001010010000001101001011100110010000001100100011010010111001101100011011010000110000101110010011001110110010101100100001000000110100101101110011101000110111100100000011110010110111101110101011100100010000001100011011000010111001001100101001000000010101100101011000011010000101000101011001010110010000001000110011010010110011101101000011101000010000001110111011010010111010001101000001000000111010001101000011010010111001100100000011011010110000101100011011010000110100101101110011001010010110000100000011000010110111001100100001000000110011101110101011000010111001001100100001000000110100101110100001000000110011001110010011011110110110100100000011101000110100001100101001000000111001101101000011000010110110101100101001000000110111101100110001000000110010001100101011001100110010101100001011101000010000000101011001010110000110100001010001010110010101100100000010100110110010101110010011101100110010100100000011101000110100001101001011100110010000001101101011000010110001101101000011010010110111001100101001011000010000001100001011100110010000001111001011011110111010100100000011101110110111101110101011011000110010000100000011010000110000101110110011001010010000001100110011010010110011101101000011101000010000001101001011101000010000001100110011011110111001000100000011110010110111101110101001000000010101100101011\\n<- EOD\\nSOCFLUSH\\n";
+							var target_id = "target";
+							var form_id = "form";
+							var input_id = "input";
+							var delay=5;
```

Comment:
Can you remove all this for me? I know it was copypasted from their file but if you're not using the huge tracts of ~~land~~ ~~shrktits~~ comments then can you just remove them from yours